### PR TITLE
Structural Health Monitoring System Challenge

### PR DESCRIPTION
**Pull request by cristianl1251**

### CODE REVIEW

File: SHM_instrumentation.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+You work in a construction company which is currently projecting a
+footbridge. The structure is supposed to be evaluated periodically
+for SHM. The Mechanical Engineering team has projected the structure 
+to have vibration frequency in a band from 2 to 50 Hz when excited.
+They have also estimated that structure vibration levels generated by
+pedestrians or any other natural phenomena are up to 60 m/s². Also, 
+your colleagues have estimated that for further data analysis, the 
+data needs to be recorded with a minimum resolution of 25 mm/s². 
+Periodically recordings will be perfomed every two months using a 
+laptop for data collection over a 4 hours long period. Considering
```
Comment (acburigo) [inline SHM_instrumentation.md]: `Recordings will be performed every two months using a laptop for data collection over a 4 hours long period.`
I don't think this sentence is needed.

File: SHM_instrumentation.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+footbridge. The structure is supposed to be evaluated periodically
+for SHM. The Mechanical Engineering team has projected the structure 
+to have vibration frequency in a band from 2 to 50 Hz when excited.
+They have also estimated that structure vibration levels generated by
+pedestrians or any other natural phenomena are up to 60 m/s². Also, 
+your colleagues have estimated that for further data analysis, the 
+data needs to be recorded with a minimum resolution of 25 mm/s². 
+Periodically recordings will be perfomed every two months using a 
+laptop for data collection over a 4 hours long period. Considering 
+the specification, your task is to design the system to record the
```
Comment (acburigo) [inline SHM_instrumentation.md]: `Considering the specification` -> `Given this specification`.

**Comment by acburigo** [comment]
I suggest changing the file name to `hw-structural_health_monitoring_system.md`.

**Comment by leoheck** [comment]
My changes, for discussion.

> You work in a construction company which is currently projecting a footbridge. The structure is supposed to be evaluated periodically for SHM. Engineers expect the structure to have a vibration frequency range from 2 to 50 Hz when excited by pedestrians or any other natural phenomena with velocities up to 60 m/s². Also, it was estimated that, for a reliable data analysis, data needs to be recorded with a minimum resolution of 25 mm/s². Given this, your task is to design a system to record data for further analysis.

File: hw-structural_health_monitoring_system.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Structural Health Monitoring System
+
+Structural Health Monitoring (SHM) is the process of evaluating the 
+health of a structure by reading an array of sensors attached to it. 
+SHM systems are, usually, installed permanently into the structure 
+for periodical evaluation of the construction. Through appropriate
+data processing and interpretation of the dynamic response measurements,
+it is possible to assess the current state of structural health,
```
Comment (leoheck) [inline hw-structural_health_monitoring_system.md]: i think this comma can be removed "health,"

**Pull request merged**